app/models/unpublishing.rb

```diff
@@ -14,8 +16,22 @@ class Unpublishing < ApplicationRecord
   validates :edition, presence: true, uniqueness: true
   validates :type, presence: true, inclusion: { in: VALID_TYPES }
   validates :explanation, presence: true, if: :withdrawal?
-  validates :alternative_path, presence: true, if: :redirect?
-  validates_with UnpublishingRedirectValidator
+  validates :redirects, presence: true, if: :redirect?
+  validates_with RoutesAndRedirectsValidator, document_type: :type, if: :redirect?
```

Comment:
may as well just have "redirect" rather than call the :type method. Can we pass in base_path too to save needing the delegate method around once alternative_paths are migrated to redirects?